website/docusaurus.config.js

```diff
@@ -179,6 +179,10 @@ module.exports = {
                         from: '/advanced/audit_log',
                         to: '/reference/event-log',
                     },
+                    {
+                        from: '/api/open_api',
+                        to: '/reference/api/unleash',
+                    },
```

Comment:
I deleted an old (unlisted) OpenAPI doc file, so I added a redirect in case anyone has any old links lying around.